tests/test.bkl

```diff
@@ -364,7 +365,8 @@
     <wx-data id="data-image-sample">
         <!-- test data for image/image.cpp test unit: -->
         <files>horse.ani  horse.bmp  horse.cur  horse.gif  horse.ico  horse.jpg
-               horse.pcx  horse.png  horse.pnm  horse.tga  horse.tif  horse.xpm</files>
+               horse.pcx  horse.png  horse.pnm  horse.tga  horse.tif  horse.xpm
+               horse.webp</files>
```

Comment:
I know you said you did not modify all the build files except for CMake, but perhaps you could still add `horse.webp` also to `samples/image/image.bkl`...